src/hott/01-paths.rzk.md

```diff
@@ -301,15 +301,15 @@ Prewhiskering paths of paths is much harder.
 ```rzk
 #def concat-concat'
   ( p : x = y)
-  : ( q : y = z) →
-    ( concat A x y z p q) = (concat' A x y z p q)
+  : ( q : y = z)
+  → ( concat A x y z p q) = (concat' A x y z p q)
   :=
     ind-path
       ( A)
       ( x)
       ( \ y' p' →
-          (q' : y' =_{A} z) →
-          (concat A x y' z p' q') =_{x =_{A} z} concat' A x y' z p' q')
+          ( q' : y' =_{A} z)
+        → ( concat A x y' z p' q') =_{x =_{A} z} concat' A x y' z p' q')
```

Comment:
Hmm, this looks a little strange to me with how the placement of `→` is inconsistent between lambdas and Pis, but this is indeed the convention. It's up to the larger community if they want to change the convention though.